### PR TITLE
Progress bars fixes

### PR DESCRIPTION
**Pull request by iandrewt**
I'll fix the documentation in the morning

Also brings a portion of `cpu_shorthand` into this branch, shouldn't cause issues? Need to set defaults too

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -881,6 +881,46 @@ getcpu () {
...
+            cpu=${cpu/Core? Duo }
+            cpu=${cpu/AMD }
+
+            case "$cpu_shorthand" in
+                "tiny") cpu=${cpu/@*} ;;
+            esac
+        ;;
+    esac
+
+    # Add CPU info bar
```
Comment (dylanaraps) [inline neofetch]: This should be:

```sh
# Add CPU info bar
prin "${subtitle}: ${cpu}"
 
cpu_usage="$(ps aux | awk 'BEGIN { sum = 0 }  { sum += $3 }; END { print sum }')"
cpu_usage="${cpu_usage/\.*}%"
 
case "$cpu_info_bar" in
    "info") prin "${subtitle} Usage: ${cpu_usage}" ;;
    "bar") prin "${subtitle} Usage: $(bar "${cpu_usage/'%'}" 100)" ;;
    "infobar") prin "${subtitle} Usage: ${cpu_usage} $(bar "${cpu_usage/'%'}" 100)" ;;
    "barinfo") prin "${subtitle} Usage: $(bar "${cpu_usage/'%'}" 100) ${cpu_usage}" ;;
esac
 
unset cpu
```

File: neofetch [hunk 2]
```diff
@@ -1398,17 +1437,22 @@ getdisk () {
...
+                "infobar") disk+=" $(bar "${disk_used/'.'}" "${disk_total/'.'}")" ;;
+                "barinfo") disk="$(bar "${disk_used/'.'}" "${disk_total/'.'}") $disk" ;;
+            esac
 
-        case "$disk_info" in
-            "off") disk="$(bar "${disk_used/'.'}" "${disk_total/'.'}")" ;;
-            *) disk+=" $(bar "${disk_used/'.'}" "${disk_total/'.'}")" ;;
-        esac
-    fi
+            ;;
```
Comment (dylanaraps) [inline neofetch]: This should only be 1 tab, not 2.

**Comment by dylanaraps** [comment]
Also, didn't we agree on changing the flags to `X_display` instead of `X_info_bar`?

File: neofetch [hunk 3]
```diff
@@ -1398,17 +1437,22 @@ getdisk () {
     # Put it all together
     disk="${disk_used} / ${disk_total} (${disk_total_per})"
 
-    if [ "$disk_bar" == "on" ]; then
-        disk_used=${disk_used/G}
-        disk_used=${disk_used/T}
-        disk_total=${disk_total/G}
-        disk_total=${disk_total/T}
+    # Add info bar
```
Comment (dylanaraps) [inline neofetch]: This should be:

```sh
    disk_used=${disk_used/G}
    disk_used=${disk_used/T}
    disk_total=${disk_total/G}
    disk_total=${disk_total/T}
 
    case "$disk_info_bar" in
        "bar") disk="$(bar "${disk_used/'.'}" "${disk_total/'.'}")" ;;
        "infobar") disk+=" $(bar "${disk_used/'.'}" "${disk_total/'.'}")" ;;
        "barinfo") disk="$(bar "${disk_used/'.'}" "${disk_total/'.'}") $disk" ;;
    esac
```

**Comment by dylanaraps** [comment]
Thanks again. :)

**Comment by iandrewt** [comment]
All fixed! Apparently I write my best code at midnight when I'm about to fall asleep :P

**Comment by dylanaraps** [comment]
All good! :+1:

**Pull request merged**